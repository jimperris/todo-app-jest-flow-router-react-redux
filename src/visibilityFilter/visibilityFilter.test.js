import VisibilityFilter from './visibilityFilter.jsx'
import {mount} from 'enzyme'
import React from 'react'
describe('reducer', () => {
  const wrap = mount(
    <VisibilityFilter filter={'SHOW_COMPLETED'} />
  )
  it('should highlight the selected filter', () => {
    expect(wrap.find('.showCompleted').prop('style')).toEqual({color: 'blue'})
    expect(wrap.find('.showAll').prop('style')).toEqual({color: 'black'})
  })
  it('should match snapshot', () => {
    expect(wrap).toMatchSnapshot()
  })
})
