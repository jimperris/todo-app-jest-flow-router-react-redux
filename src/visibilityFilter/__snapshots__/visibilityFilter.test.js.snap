// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducer should match snapshot 1`] = `
<div>
  <button
    className="showAll"
    onClick={[Function]}
    style={
      Object {
        "color": "black",
      }
    }
  >
    Show All
  </button>
  <button
    className="showCompleted"
    onClick={[Function]}
    style={
      Object {
        "color": "blue",
      }
    }
  >
    Show Completed
  </button>
  <button
    className="showInProgress"
    onClick={[Function]}
    style={
      Object {
        "color": "black",
      }
    }
  >
    Show In Progress
  </button>
</div>
`;
